File: problems/a/apaxianparent/apaxianparent.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    y, p = input().split()

    if y[-2:] == "ex":
        print(y+p)
    elif y[-1] == 'e':
        print(y+'x'+p)
    elif y[-1] in "aiou":
        print(y[:-1]+"ex"+p)
    else:
        print(y+'ex'+p)


if __name__ == '__main__':
    main()

File: problems/a/apaxianparent/apaxianparent.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <bits/stdc++.h>

using namespace std;

int main() {
    string y, p, y_end;

    cin >> y >> p;

    y_end = y.substr(y.size()-2, 2);

    if (y_end == "ex") {
        cout << y << p << endl;
    }
    else if (y_end[1] == 'e') {
        cout << y << 'x' << p << endl;
    }
    else if (y_end[1] == 'a' || y_end[1] == 'i' || y_end[1] == 'o' || y_end[1] == 'u') {
        cout << y.substr(0, y.size() - 1) << "ex" << p << endl;
    }
    else {
        cout << y << "ex" << p << endl;
    }
}